perl-lib/OESS/t/Interface_test.t

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/perl  -T
+
+use OESS::Interface;
+use Test::More tests => 30;
+use Log::Log4perl;
+use OESS::DB;
+use Data::Dumper;
+use Test::Deep;
+
+
+# Initialize logging
+Log::Log4perl->init("/etc/oess/logging.conf");
+
+# Initialize DB
+my $db = OESS::DB->new();
```

Comment:
we need to make this use the test config file not the one in /etc/oess/database.xml

adding the following to the use statements will help

use FindBin;
my $path;

BEGIN {
    if($FindBin::Bin =~ /(.*)/){
            $path = $1;
      }
}

use lib "$path";
use OESSDatabaseTester;

then you can do 

my $db = OESS::DB->new(config => OESSDatabaseTester::getConfigFilePath());

basically replace /etc/oess/database.xml with `OESSDatabaseTester::getConfigFilePath()` everywhere